x-pack/plugins/ml/public/application/model_management/test_models/test_trained_models_context.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createContext, Dispatch, useContext } from 'react';
+import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+
+export interface TestTrainedModelsContextType {
+  pipelineConfig: estypes.IngestPipeline | undefined;
+  createPipelineFlyoutOpen: boolean;
+}
+export const TestTrainedModelsContext = createContext<
+  | {
+      currentContext: TestTrainedModelsContextType;
+      setCurrentContext: Dispatch<TestTrainedModelsContextType>;
+    }
+  | undefined
+>(undefined);
+
+export function useTestTrainedModelsContext() {
+  const testTraindedModelsContext = useContext(TestTrainedModelsContext);
```

Comment:
typo `Trainded`

Reply:
Good catch- fixed in c582e99a7ece7292c11667215cdffd931c2b8a6a